src/action-bar.scss

```diff
@@ -43,25 +41,47 @@ $block: #{$fd-namespace}-action-bar;
     }
   }
 
+  @include fd-screen(s) {
+    padding-left: map_get($fd-action-bar-padding-x, "s");
```

Comment:
where are you getting these measurements

Reply:
All these measurements based on the fiori 3 standards and here is the new design prepared based on the fiori 3 specs like typography spacings etc.
https://sapariba.invisionapp.com/share/7WSNCMKJQHP#/screens/370170777_ActionBar